python/GangaDirac/Lib/Server/DiracCommands.py

```diff
@@ -58,6 +58,11 @@ def getReplicas(lfns):
     output(dirac.getReplicas(lfns, active=True, preferDisk = True))
 
 
+def getAccessURL(lfn,SE):
+    ''' Return the access URL for the given LFN and storage element '''
+    output(dirac.getAccessURL(lfn,SE))
```

Comment:
These functions are run in a special DIRAC Python process which defines the `output` function so that it pushes its argument to the stdout (or maybe some other pipe, I can't remember). The process which originally called the function consumes this pipe and so get the string representation of the `output` argument which it then `eval`s.

Confusing I agree but the best solution given the constraints.